packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -324,41 +322,44 @@ const Banner = (props: Props): React.ReactElement => {
 const bannerTokens = {
     root: {
         border: {
-            radius: {
-                default: border.radius.radius_0,
-                floating: border.radius.radius_040,
-            },
+            radius: theme.root.border.radius,
```

Comment:
All the banner token values that are replaced with `theme.*` means the value is different in TB (and we can't use semantic tokens to address the difference)! Let me know if you have any feedback on this approach

Reply:
that's the way I've approached this as well :)